tests/phpunit/mockers/class-wc-mocker-amazon-payments-advanced-api.php

```diff
@@ -51,4 +51,24 @@ public static function get_create_checkout_classic_session_config( array $payloa
 			'signature'   => $signature,
 		);
 	}
+
+	public static function is_region_supports_shop_currency( $region, $currency = false ) : bool {
```

Comment:
Why here we take as param the region instead of figuring it out within the method like done in production code ?

Reply:
Since this is a mocker class to make it simpler and avoid using the original class, so everything is wrapped in the mocker.